darwin/Platform.c

```diff
@@ -15,6 +15,10 @@ in the source distribution for its full text.
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/_types/_mach_port_t.h>
+#include <sys/socket.h>
+#include <net/if.h>
+#include <net/if_types.h>
+#include <net/route.h>
```

Comment:
What do you suggest?

Reply:
Have a look at the style guide. Alphabetical, parent directory first, system headers before project headers (exception: associated file header).

Reply:
OK. I didn't know alphabetical order is required since this requirement is missing from the style guide.

Reply:
I'm certain it's in the style guide. But given some systems have broken headers, it's worded less strictly (alphabetical, unless required otherwise, but add comment in that case).